verisignmdns/README.md

```diff
@@ -0,0 +1,9 @@
+Verisign MDNS Provider enables access to Verisign MDNS through Denominator. This provider lets you perform zone list and resource record set management either using Denominator cli or through java API.
+
+This provider currently supports 'put', 'iterator', 'iterate by name and type', 'get by name and type', and  'delete by name and type' operations for CNAME, A, AAAA, NS, NAPTR, PTR, MX, DS, SRV and TXT record types. 
+
+Note - 
```

Comment:
prefer markdown to call out headings and list items https://help.github.com/articles/markdown-basics/